cylc/flow/scripts/lint.py

```diff
@@ -398,6 +437,14 @@ def check_for_obsolete_environment_variables(line: str) -> List[str]:
             check_if_jinja2,
             function=re.compile(r'(?<!{)#.*?{[{%]').findall
         )
+    },
+    'S012': {
+        'short': 'placeholder',
+        FUNCTION: lambda *args, **kwargs: True
+    },
```

Comment:
Eh?

Reply:
Stops future developers accidentally over-writing S012 which is reserved for line length check. I've added a bit more explanation, and removed the FUNCTION, because it's unnecessary.

Reply:
Ah right, if you could jam in a comment to clarify.

Reply:
I've changed short to make it clearer.